packages/ses/src/error/assert.js

```diff
@@ -375,6 +375,9 @@ const makeAssert = (optRaise = undefined, unredacted = false) => {
   };
   freeze(fail);
 
+  /** @type {FailTag} */
```

Comment:
Will anything else implement FailTag? If not I think we're better off inlining the type here and when the type of this needs to be referenced using "typeof"

Reply:
But assert.js imports types.js, and types.js uses `FailTag`, but if we inlined it, we'd inline it in assert.js. Advice? How do you want this layered?

Reply:
Ah. Transitively we need an interface, `Assert`, so this is appropriate

Reply:
> so this is appropriate

Which "this"? Do you mean that the status quo FailTag of this PR is fine as is?

Reply:
status quo

Reply:
Cool. Resolving. Thanks.